tests/scripts/thread-cert/message.py

```diff
@@ -294,8 +306,30 @@ def next_coap_message(self, code, uri_path=None, assert_enabled=True):
 
         return message
 
+    #Get the newest mle message
```

Comment:
Would it be better to update the comment as below?
> Get the last Mle Message with specified type from existing capture.

Reply:
fixed